hms-app/src/pages/Home.tsx

```diff
@@ -22,6 +23,7 @@ function Home() {
     openDate: new Date(),
     closeDate: new Date(),
   })
+  const [value, onChange] = useState('')
```

Comment:
do you ever use `value`?

Reply:
no i dont because its read only but i had to define it that way